.bazelrc

```diff
@@ -28,19 +34,17 @@ build:nvcc --@rules_cuda//cuda:compiler=nvcc
 build:nvcc --repo_env=BAZEL_CXXOPTS=  # Prevent warning: 'std' redefinition.
 build:nvcc --@rules_cuda//cuda:copts=--expt-relaxed-constexpr
 
-# Disable warnings we don't care about.
-build --copt=-Wno-unused-local-typedef
-
 # C++14 standard version is required.
-build --cxxopt=-std=c++14 --host_cxxopt=-std=c++14
-
-# Necessary linker options.
-build --linkopt=-pthread
-build --linkopt=-ldl
-build --linkopt=-lm
-build --host_linkopt=-pthread
-build --host_linkopt=-ldl
-build --host_linkopt=-lm
+build:android --cxxopt=-std=c++14
```

Comment:
I agree, per-toolchain is better than per-platform. 

As a side note - today Windows is assumed to mean MSVC, but it may be worth considering a Windows build with Clang.

Reply:
Yes, we are very much considering it. One issue is that CUDA clang on Windows is not officially supported.